scripts/vnet_route_check.py

```diff
@@ -219,15 +219,24 @@ def get_vnet_routes_from_app_db():
             vnet_routes[vnet_name] = {}
             vnet_routes[vnet_name]['routes'] = []
 
-            intf = vnet_intfs[vnet_name][0]
-            vnet_routes[vnet_name]['vrf_oid'] = vnet_vrfs.get(intf, 'None')
+            if vnet_name not in vnet_intfs:
+                # this route has no vnet_intf and may be part of default VRF.
+                vnet_table = swsscommon.Table(db, 'VNET_TABLE')
+                scope = vnet_table.get(vnet_name)[1][1][1]
```

Comment:
what is this index [1][1][1] ? Can you provide an example?

Reply:
this looking up the scope=default but i think I can improve it.